File: backend/back-pasteleria/src/main/java/cl/milsabores/pasteleria/dto/ProductResponse.java
package cl.milsabores.pasteleria.dto;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class ProductResponse {
    String codigo;
    String categoria;
    String nombre;
    Integer precio;
    String descripcion;
    Boolean popular;
    String historia;
    String imagenUrl;
}

File: app/src/main/java/com/example/pasteleriamilssaboresandroid/data/network/dto/ProductResponse.kt
package com.example.pasteleriamilssaboresandroid.data.network.dto

data class ProductResponse(
    val codigo: String,
    val categoria: String,
    val nombre: String,
    val precio: Int,
    val descripcion: String,
    val popular: Boolean,
    val historia: String,
    val imagenUrl: String
)

